standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/StandardizationRerunSuite.scala

```diff
@@ -0,0 +1,165 @@
+
+class StandardizationRerunSuite extends fixture.FunSuite with SparkTestBase with TempFileFixture {
+
+  import za.co.absa.enceladus.utils.implicits.DataFrameImplicits.DataFrameEnhancements
+
+  private implicit val udfLib: UDFLibrary = new UDFLibrary
+
+  private val tmpDirPrefix = "StdRerunTest"
+  private val tmpFilePrefix = "test-input-"
+  private val tmpFileSuffix = ".csv"
+
+  private val csvContent: String =
+    """101|102|1|2019-05-04|2019-05-04
+      |201|202|2|2019-05-05|2019-05-05
+      |301|302|1|2019-05-06|2019-05-06
+      |401|402|1|2019-05-07|2019-05-07
+      |501|502||2019-05-08|2019-05-08"""
+      .stripMargin
+
+  private val dataSet = Dataset("SpecialColumns", 1, None, "", "", "SpecialColumns", 1, conformance = Nil)
+
+  type FixtureParam = String
+
+  def withFixture(test: OneArgTest): Outcome = {
+    val tmpFile = createTempFile(tmpFilePrefix, tmpFileSuffix, StandardCharsets.UTF_8, csvContent)
+    val outcome = test(tmpFile.getAbsolutePath)
+    tmpFile.delete()
+    outcome
+  }
+
+  /** Creates a dataframe from an input file name path and command line arguments to Standardization */
+  private def getTestDataFrame(tmpFileName: String, schemaStr: StructType): DataFrame = {
+    val args = ("--dataset-name SpecialColumns --dataset-version 1 --report-date 2019-07-23 " +
+      "--report-version 1 --raw-format csv --header false --delimiter |").split(" ")
+
+    val cmd: CmdConfig = CmdConfig.getCmdLineArguments(args)
+    StandardizationJob
+      .getFormatSpecificReader(cmd, dataSet)
+      .schema(schemaStr)
+      .load(tmpFileName)
+  }
+
+  test("Test standardizing a CSV without special columns") { tmpFileName =>
+    val schema: StructType = StructType(Seq(
+      StructField("A1", IntegerType, nullable = true),
+      StructField("A2", IntegerType, nullable = true),
+      StructField("A3", IntegerType, nullable = true),
+      StructField("A4", StringType, nullable = true,
+        Metadata.fromJson("""{"pattern": "yyyy-MM-dd"}""")),
+      StructField("A5", StringType, nullable = true)
+    ))
+
+    val schemaStr: StructType = StructType(Seq(
+      StructField("A1", StringType, nullable = true),
+      StructField("A2", StringType, nullable = true),
+      StructField("A3", StringType, nullable = true),
+      StructField("A4", StringType, nullable = true),
+      StructField("A5", StringType, nullable = true)
+    ))
+
+    val expectedOutput =
+      """+---+---+----+----------+----------+------+
+        ||A1 |A2 |A3  |A4        |A5        |errCol|
+        |+---+---+----+----------+----------+------+
+        ||101|102|1   |2019-05-04|2019-05-04|[]    |
+        ||201|202|2   |2019-05-05|2019-05-05|[]    |
+        ||301|302|1   |2019-05-06|2019-05-06|[]    |
+        ||401|402|1   |2019-05-07|2019-05-07|[]    |
+        ||501|502|null|2019-05-08|2019-05-08|[]    |
+        |+---+---+----+----------+----------+------+
+        |
+        |""".stripMargin.replace("\r\n", "\n")
```

Comment:
@benedeki please run these tests.
I am curious how it works if the `csvContent` file was created with system line endings while when read we replace CRLF for LF

Reply:
Looking into it...